src/odemis/driver/test/autoscript_client_test.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on 10 Dec 2024
+
+Copyright © 2024 Delmic
+
+This file is part of Delmic Acquisition Software.
```

Comment:
Could you adjust the header to say "Odemis" instead of "Delmic Acquisition Software" (that was the very early name!)